guava/src/com/google/common/math/PairedStatsAccumulator.java

```diff
@@ -239,9 +239,6 @@ private static double ensureInUnitRange(double value) {
     if (value >= 1.0) {
       return 1.0;
     }
-    if (value <= -1.0) {
-      return -1.0;
-    }
-    return value;
+    else return Math.max(value, -1.0);
```

Comment:
Could this whole method use `Doubles.constrainToRange()`?

That would be a more simplifying change.

Reply:
Really clean and strong change, i will replace both with `constrainToRange()`